packages/front-end/components/SavedGroups/ConditionGroups.tsx

```diff
@@ -150,12 +164,27 @@ export default function ConditionGroups({ groups, mutate }: Props) {
                           />
                         </td>
                         <td>{truncateString(s.description || "", 40)}</td>
+                        <td>
+                          {(s?.projects?.length || 0) > 0 ? (
+                            <ProjectBadges
+                              resourceType="saved group"
+                              projectIds={s.projects}
+                              className="badge-ellipsis short align-middle"
+                            />
+                          ) : (
+                            <ProjectBadges
+                              resourceType="saved group"
+                              className="badge-ellipsis short align-middle"
+                            />
+                          )}
+                        </td>
                         <td>{s.owner}</td>
                         <td>{ago(s.dateUpdated)}</td>
-                        {canUpdate || canDelete ? (
+                        {canUpdate({ projects: s.projects }) ||
+                        canDelete({ projects: s.projects }) ? (
```

Comment:
For all of these permission methods, just pass in the saved group interface in its entirety instead of creating an object with just `projects`.

For example, do this:

```ts
canUpdate(s) || canDelete(s) ? ...
```